### PR TITLE
Edit and delete Topic

### PR DESCRIPTION
**Pull request by Pablo-Mdz**
---
name: Pull Request Template
about: Edit Topic done and Delete Topic in progress
---

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no new functionality, just code cleanup/refactoring)
- [ ] Documentation update

## Testing Steps

Not finished!!!
I’m working on Delete function.

### CODE REVIEW

**Comment by Pablo-Mdz** [comment]
Now is working the edit and Delete function.

But there are a bug : 

Screenshot 2023-04-06 at 17 55 41

I think that is on Delete function, but I see it ok. 
Despite this warning, the functions work perfectly.

File: client/src/components/Topic/TopicDetails.jsx [hunk 1]
```diff
@@ -54,17 +60,15 @@ export default function TopicDetails() {
                 onCancel={cancelEditing}
               />
             ) : (
-              <PostCard post={post} onEdit={handleEdit} />
+              <PostCard className="py-2" post={post} onEdit={handleEditPost} />
             )}
```
Comment (antoniosaleme) [inline client/src/components/Topic/TopicDetails.jsx]: not sure if these className="py-2" is working. If it is not working, remove it.

File: client/src/components/Topic/TopicDetails.jsx [hunk 2]
```diff
@@ -54,17 +60,15 @@ export default function TopicDetails() {
...
-            <div className="p-6">
-              <p className="text-gray-800 text-base">Date: {post.createdAt}</p>
-            </div>
+            {/* <div className="p-6">
+              <p className="text-gray-800 text-base">Date: {new Date(post.createdAt).toLocaleDateString('es-ES')}</p>
+            </div> */}
```
Comment (antoniosaleme) [inline client/src/components/Topic/TopicDetails.jsx]: if possible, remove it

**Review by antoniosaleme — APPROVED** [review]
just small changes, then you can merge

**Pull request merged**